### ISSUE TITLE
Build on M1 / osx-arm64

### ISSUE THREAD

**Issue by mattwthompson** [conda-forge/ambertools-feedstock#77]
Upstreams done! Bot PR here: https://github.com/conda-forge/ambertools-feedstock/pull/80

1. `parmed`: https://github.com/conda-forge/parmed-feedstock/pull/22
2. `netcdf-fortran`: https://github.com/conda-forge/netcdf-fortran-feedstock/pull/67
3. `packmol`:  ~~https://github.com/conda-forge/packmol-feedstock/pull/9~~ https://github.com/conda-forge/packmol-feedstock/pull/12

**Comment by jaimergp** [comment]
If the PR doesn't arrive before tomorrow, check the status [here](https://conda-forge.org/status/#armosxaddition) because it might be a dependency or several stuck in the migration queue.

**Comment by jaimergp** [comment]
Namely, this image:

<img id="Container_armosxadditionGraph" src="https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/status/armosxaddition.svg?sanitize=true" style="display: block;">

Ambertools needs ParmEd, PackMol and netCDF-Fortran.

**Comment by mattwthompson** [comment]
Ahh I was just a few seconds behind - this is a really cool tool.

![image](https://user-images.githubusercontent.com/7935382/141510503-46dc2792-00c1-4e2a-a0b2-2c72ec103941.png)

I'll kick the tires on those to see if the bots can chew away at them.

**Comment by jaimergp** [comment]
The PRs should have been sent to those any minute now already. Btw, I was surprised you can actually Ctrl+F on a SVG image. It just needs its own tab :D

**Comment by mattwthompson** [comment]
Great! I didn't know they'd happen automatically. I'll keep an eye on them and hope for the best

I was also surprised to see how easily that graph could be searched. Really cool!

**Comment by mattwthompson** [comment]
Bots did their work - three PRs opened, one merged!

**Comment by hainm** [comment]
Happy new year to all of you. What's the status of this issue? I've seen an user trying with `osx-arm64
` too: https://github.com/Amber-MD/pytraj/issues/1591#issuecomment-1005940786

**Comment by mattwthompson** [comment]
Hey Hai,

The bots are blocked by an upstream (https://github.com/conda-forge/packmol-feedstock/pull/9) which I don't have the hardware/know-how to do myself. If that's fixed we'll see if any changes are needed here. I'm not so involved with the Amber community and mailing list so I'm not sure if anybody has tried this on their own M1 hardware yet. I have no idea if/how many changes would be needed here, or if the source code would need to be patched as well.

**Comment by hainm** [comment]
Thanks Matt for letting me know.

**Comment by mattwthompson** [comment]
A new PR to the PACKMOL feedstock is passing. If/when that's merged, the bot should soon trigger a migration attempt here.

**Comment by mikemhenry** [comment]
https://github.com/conda-forge/packmol-feedstock/pull/12 has been merged :tada:

**Comment by jaimergp** [comment]
And here is the arm64 PR https://github.com/conda-forge/ambertools-feedstock/pull/80

**Comment by jchodera** [comment]
Slowly but surely!
![https://user-images.githubusercontent.com/7935382/141510503-46dc2792-00c1-4e2a-a0b2-2c72ec103941.png](https://user-images.githubusercontent.com/7935382/141510503-46dc2792-00c1-4e2a-a0b2-2c72ec103941.png)

**Comment by jchodera** [comment]
It seems like we're almost there---do we need to finish up https://github.com/conda-forge/ambertools-feedstock/pull/85 or https://github.com/conda-forge/ambertools-feedstock/pull/80?

**Comment by mikemhenry** [comment]
Happy to close this one :)
